### PR TITLE
Executor events are not always "exited" here

### PR DESCRIPTION
**Pull request by dstandish**
@wolfier this makes more general that misleading message we encountered the other day.  (task isn't always "exiting") 

what do you think of the current messaging?

### CODE REVIEW

File: airflow/jobs/scheduler_job_runner.py [hunk 1]
```diff
@@ -705,12 +705,14 @@ def _process_executor_events(self, session: Session) -> int:
...
 
             self.log.info(
-                "Executor reports execution of %s.%s run_id=%s exited with status %s for try_number %s",
+                "Executor received event with state %s for task instance "
+                "dag_id=%s task_id=%s run_id=%s try_number=%s map_index=%s",
+                state,
                 ti_key.dag_id,
                 ti_key.task_id,
                 ti_key.run_id,
-                state,
```
Comment (uranusjr) [inline airflow/jobs/scheduler_job_runner.py:712]: Do we not want state here?

Comment (dstandish) [reply airflow/jobs/scheduler_job_runner.py:712]: we do, good catch thanks

Comment (dstandish) [reply airflow/jobs/scheduler_job_runner.py:712]: oh, i did not drop it, it was just moved up :)

File: airflow/jobs/scheduler_job_runner.py [hunk 2]
```diff
@@ -705,12 +705,14 @@ def _process_executor_events(self, session: Session) -> int:
             ti_primary_key_to_try_number_map[ti_key.primary] = ti_key.try_number
 
             self.log.info(
-                "Executor reports execution of %s.%s run_id=%s exited with status %s for try_number %s",
+                "Executor received event with state %s for task instance "
```
Comment (jedcunningham) [inline airflow/jobs/scheduler_job_runner.py]:
```suggestion
                "Executor sent event with state %s for task instance "
```

The executor is sending the event, scheduler job is receiving it.

File: airflow/jobs/scheduler_job_runner.py [hunk 3]
```diff
@@ -705,12 +705,14 @@ def _process_executor_events(self, session: Session) -> int:
...
+                "dag_id=%s task_id=%s run_id=%s try_number=%s map_index=%s",
```
Comment (jedcunningham) [inline airflow/jobs/scheduler_job_runner.py]: I wonder if we should just toss ti_key here and let its str do it's thing. Same info.

File: airflow/jobs/scheduler_job_runner.py (hunk 2 above)
Comment (uranusjr) [reply airflow/jobs/scheduler_job_runner.py:708]: Probably a good idea; it’d be easier if we add another field to the key in the future (like we did `map_index`; although of course this doesn’t happen at all often anyway)

File: airflow/jobs/scheduler_job_runner.py (hunk 3 above)
Comment (uranusjr) [reply airflow/jobs/scheduler_job_runner.py]: Probably a good idea; it’d be easier if we add another field to the key in the future (like we did map_index; although of course this doesn’t happen at all often anyway)

File: airflow/jobs/scheduler_job_runner.py (hunk 2 above)
Comment (dstandish) [reply airflow/jobs/scheduler_job_runner.py]: Maybe we should say "received executor event" then? wdyt

File: airflow/jobs/scheduler_job_runner.py (hunk 3 above)
Comment (dstandish) [reply airflow/jobs/scheduler_job_runner.py]: good with that

File: airflow/jobs/scheduler_job_runner.py (hunk 2 above)
Comment (dstandish) [reply airflow/jobs/scheduler_job_runner.py]: pushed that change fyi

**Review by jedcunningham — APPROVED** [review]

**Review by uranusjr — APPROVED** [review]

**Pull request merged**